src/Microsoft.ML.Data/Evaluators/Metrics/MultiClassClassifierMetrics.cs

```diff
@@ -57,14 +36,38 @@ public sealed class MultiClassClassifierMetrics
         /// </remarks>
         public double LogLossReduction { get; private set; }
 
+        /// <summary>
+        /// Gets the macro-average accuracy of the model.
+        /// </summary>
+        /// <remarks>
+        /// The macro-average is computed by taking the average over all the classes of the fraction
+        /// of correct predictions in this class (the number of correctly predicted instances in the class,
+        /// divided by the total number of instances in the class).
```

Comment:
Something like this:

The macro-average is the average accuracy for the classifier at the class level. The accuracy for each class is computed and these accuracies are averaged to obtain the macro accuracy.